### PR TITLE
added shipyard page

### PR DESCRIPTION
**Pull request by piyusesh**
# Description
added shipyard page

Screenshot from 2024-03-03 14-13-26

### CODE REVIEW

File: src/pages/GitHubRepoCard.jsx
Comment (codecShivam) [inline src/pages/GitHubRepoCard.jsx:1]: Everything looks good to me. Could you please make a minor alignment change at the GitHub link .Also, ensure that the repo owner details are aligned properly.
Screenshot from 2024-03-03 14-30-34

Add one more project to it : https://github.com/fOSS-Community/quickfire

great work !  @piyusesh

**Review by codecShivam — CHANGES_REQUESTED** [review]
Everything looks good to me. Could you please make a minor alignment change at the GitHub link .Also, ensure that the repo owner details are aligned properly.
Screenshot from 2024-03-03 14-30-34

Add one more project to it : https://github.com/fOSS-Community/quickfire

great work !  @piyusesh

**Comment by piyusesh** [comment]
@codecShivam alright working on it !

**Comment by codecShivam** [comment]
> @codecShivam alright working on it !

Hey @piyusesh, please mention this pull request in its related ticket so that I can assign it to you

**Comment by piyusesh** [comment]
@codecShivam #240 this is the issue

**Comment by codecShivam** [comment]
> @codecShivam #240 this is the issue

Yeah, Just drop a comment on that issue

**Pull request merged**